js/maze.js

```diff
@@ -0,0 +1,235 @@
+"use strict";
+
+function Maze(size){
+    
+    if(size === 'small'){
+        this.dimension = 15;
+        this.generationSpeed = 100;
+    }
+    if(size === 'medium'){
+        this.dimension = 25;
+        this.generationSpeed = 60;
+    }
+    if(size === 'large'){
+        this.dimension = 45;
+        this.generationSpeed = 40;
+    }
+    this.cellsArray = this.createCells();
+    this.solution = [];
+
+}
+    
+Maze.prototype.createCells = function(){
+    var count = 0;
+
+    var cellsArray = [];
+    
+    for(var i = 0; i < this.dimension; i++){
+        cellsArray[i] = [];
+        for(var j = 0; j < this.dimension; j++){
+            
+            var cell = new MazeCell(i, j);
+            cellsArray[i][j] = cell;
+        }
+    }
+
+    return cellsArray;
+
+}
+    
+Maze.prototype.showMazeField = function(cellsArray){
+    var container = document.querySelector('.mazeContainer');
+
+    for(var cellX= 0; cellX<cellsArray.length; cellX++){
+        for(var cellY = 0; cellY<cellsArray.length; cellY++){
+            
+            var element = document.createElement('div');
+            element.setAttribute('class', 'cell');
+            cellsArray[cellX][cellY].setElement(element);
+            container.appendChild(element);
+        }
```

Comment:
I think that it's better to use innerHTML instead of createElement and appendChild to render maze field, because it's easier to support and change.